client/README.md

```diff
@@ -1,70 +1,16 @@
-# Getting Started with Create React App
+# Traveler's Tea
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Running the app
 
-## Available Scripts
+`yarn install` to install any dependencies
 
-In the project directory, you can run:
+`cd client; yarn start` to start the app on [http://localhost:3000](http://localhost:3000)
```

Comment:
We should installation and building instructions to the root in the future. For the grading rn let's keep it here.

Reply:
We also need install and build instructions for server.

Adding to issues